test/libsolidity/syntaxTests/lvalues/functions.sol

```diff
@@ -12,4 +12,3 @@ contract C {
 }
 // ----
 // TypeError: (83-84): Expression has to be an lvalue.
-// TypeError: (166-172): Expression has to be an lvalue.
```

Comment:
Why is this error gone? If it is because of a new fatal error, can you create a new test that contains the second half?

Reply:
The first one is fatal. But yes, I will create a test file for the second one.